Formula/jsmn.rb

```diff
@@ -0,0 +1,92 @@
+class Jsmn < Formula
+  desc "World fastest JSON parser/tokenizer"
+  homepage "https://zserge.com/jsmn/"
+  url "https://github.com/zserge/jsmn/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "5f0913a10657fe7ec8d5794ccf00a01000e3e1f2f1e1f143c34a0f7b47edcb38"
+  license "MIT"
+
+  def install
+    system "make"
+    include.install "jsmn.h"
```

Comment:
What does `make` do if we just end up installing a single header file?

Presumably the header is generated by `make`, but it seems weird to have to do that at all...

Reply:
I was following out-of-date instructions. Looking at the current version of the Makefile, it turns out that running `make` runs the tests. There's no reason to call it from the formula.